language-support/java/codegen/BUILD.bazel

```diff
@@ -104,7 +104,6 @@ pom_file(
 )
 
 daml_lf_target_versions = [
-    "1.0",
```

Comment:
@stephencompall-DA @leo-da You might be able to remove some code due to this change.

Reply:
Almost certainly not, but thanks for thinking of us 😀